misc/lldb_disasm.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+#coding: utf-8
+#
+# Usage: run `command script import -r misc/lldb_disasm.py` on LLDB
+#
+#
+#    (lldb) p iseq
+#    (rb_iseq_t *) $147 = 0x0000000101068400
+#    (lldb) rbdisasm iseq
+#    0000 putspecialobject( 3 )
+#    0002 putnil
+#    0003 defineclass( ID: 0x560b, (rb_iseq_t *)0x1010681d0, 2 )
+#    0007 pop
+#    0008 putspecialobject( 3 )
+#    0010 putnil
+#    0011 defineclass( ID: 0x56eb, (rb_iseq_t *)0x101063b58, 2 )
+#    0015 leave
+
+
+import lldb
+import os
+import shlex
+
+TS_VARIABLE = b'.'[0]
+TS_CALLDATA = b'C'[0]
+TS_CDHASH   = b'H'[0]
+TS_IC       = b'K'[0]
+TS_IVC      = b'A'[0]
+TS_ID       = b'I'[0]
+TS_ISE      = b'T'[0]
+TS_ISEQ     = b'S'[0]
+TS_OFFSET   = b'O'[0]
+TS_VALUE    = b'V'[0]
+TS_LINDEX   = b'L'[0]
+TS_FUNCPTR  = b'F'[0]
+TS_NUM      = b'N'[0]
+TS_BUILTIN  = b'R'[0]
+
+def rb_vm_insn_addr2insn2(target, result, wanted_addr, size):
+    tIntPtr = target.FindFirstType("intptr_t")
+    sizeOfIntPtr = tIntPtr.GetByteSize();
+    addr_of_table = target.FindSymbols("vm_exec_core.insns_address_table")[0].GetSymbol().GetStartAddress().GetLoadAddress(target)
+
+    for insn in range(0, size):
```

Comment:
`range(0, size)` -> could be `range(size)`, `0` redundant here

https://docs.python.org/3.8/library/functions.html#func-range